lib/rubocop/cop/mixin/range_help.rb

```diff
@@ -51,9 +51,18 @@ def range_with_surrounding_comma(range, side = :both)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
-      def range_with_surrounding_space(range:, side: :both,
-                                       newlines: true, whitespace: false,
-                                       continuations: false)
+      NOT_GIVEN = Module.new
+      def range_with_surrounding_space(range_positional = NOT_GIVEN, # rubocop:disable Metrics/ParameterLists
+                                       range: NOT_GIVEN, side: :both, newlines: true,
+                                       whitespace: false, continuations: false)
+        unless range == NOT_GIVEN
+          warn <<~WARNING
+            Passing `range` as a keyword argument is deprecated, and will be removed in RuboCop 2.0. Pass the range as the first positional argument instead.
```

Comment:
Frankly, I don't see a good reason to ever drop this given how tiny this is. My time with Clojure has made me extra apprehensive of doing breaking changes for no good reason. We can remove this in 3.0 or 4.0 to give more time to people to migrate. Let's make this message less specific.

Reply:
Yeah, as mentioned in https://github.com/rubocop/rubocop/pull/10727#discussion_r902014511. I think this change can start by not displaying the warning message. This is a non-essential warning for many end users.

> We can remove this in 3.0 or 4.0 to give more time to people to migrate.

I also like to see this change in a long migration path :-)